known-issues.rst

```diff
@@ -16,7 +16,10 @@ Binary installation issues
 Cross Platform
 --------------
 
-*No known issues.*
+- The installation is only tested with the Conda environment provided by the <newinstall.sh> 
+  or <lsstsw> installation methods. You may attempt to install ``lsst_distrib`` without this 
+  Conda environment provided that you also install the packages listed in :ref:`prereq-platforms`, 
+  though these installations may not behave as expected. See :jira:`DM-15040` and :jira:`DM-15041`.
```

Comment:
I don't think @jonathansick will approve of the hard line wrapping (& I hope he'll forgive me if I'm wrong). Also, it looks like there is some trailing whitespace on these lines.

Other than that, I don't love the above wording, but I think it's ok given the apparent urgency of this release. I hope we can do better next time.

Reply:
How about this?

```
- This release may not work when installed without the Conda environments provided by the documented :doc:`newinstall.sh </install/newinstall>`, :doc:`lsstsw </install/lsstsw>`, or :doc:`Docker <docker>` installation methods.
  The :ref:`documented Python dependencies <python-deps>` may differ slightly from those in the release, causing issues with the :doc:`validation demo </install/demo>`.
  See :jira:`DM-15040` and :jira:`DM-15041` for more discussion.
```

Does this capture the issue appropriately?